docs/clusterfuzzlite.md

```diff
@@ -11,23 +11,38 @@ permalink: /
 ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
 ClusterFuzzLite is based on [ClusterFuzz].
 
-By fuzzing your code, ClusterFuzzLite can make it more secure as fuzzing is
-highly effective technique for finding bugs in software.
-
-ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build].
+Fuzzing is a highly effective technique for finding bugs in software.
+ClusterFuzzLite makes your code more secure by fuzzing changes before they even
+enter the codebase.
+ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build], getting the
+benefits of continuous fuzzing is simple if you can use these CI systems.
 Suppport for more CI systems is in-progress and
 [extending support to other CI systems] is easy.
 
-ClusterFuzzLite offers features that will make fuzzing an important part of your
-software development lifecycle, including:
+ClusterFuzzLite offers many useful features, including:
 1. Quickly fuzzing code changes (pull requests) before they land and allowing
```

Comment:
Change to unordered lists (bullet points) since these are examples, not steps that have to happen in order